MSAL/IdentityCore

```diff
@@ -1 +1 @@
-Subproject commit 525cfb35eb7ea82ca6b9e1cfc1af2bf403bbf39f
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->